koku/masu/test/external/downloader/gcp_local/test_gcp_local_report_downloader.py

```diff
@@ -93,7 +92,9 @@ def test_initializer(self):
         """Test the GCP-Local initializer."""
         self.assertIsNotNone(self.report_downloader)
 
-    def test_download_file(self):
+    @patch("masu.util.aws.common.remove_files_not_in_set_from_s3_bucket")
+    @patch("masu.util.aws.common.copy_data_to_s3_bucket")
+    def test_download_file(self, _, __):
```

Comment:
Tritto (yes, I just made that up).
```suggestion
    def test_download_file(self, *args):
```